internal/pipe/checksums/checksums_test.go

```diff
@@ -304,47 +314,87 @@ func TestPipeCheckSumsWithExtraFiles(t *testing.T) {
 			},
 		},
 	}
+	checksumsMap := map[string]string{
+		"crc32":  "f94d3859",
+		"md5":    "5ac749fbeec93607fc28d666be85e73a",
+		"sha1":   "8b45e4bd1c6acb88bebf6407d16205f567e62a3e",
+		"sha224": "21bc225587d8768058837b68fe7e0341e87b972f02fd8fb0c236d1d3",
+		"sha256": "61d034473102d7dac305902770471fd50f4c5b26f6831a56dd90b5184b3c30fc",
+		"sha384": "f6055a96a105d2fb5941a616964ffda8294fd415730cc4154a602062bc3d00e99d3c6f4a11af8c965a343de4afca3c2b",
+		"sha512": "14925e01a7a0cf0801aa95fe52d542b578af58ae7997ada66db3a6eae68a329d50600a5b7b442eabf4ea77ea8ef5fe40acf2ab31d47311b2a232c4f64009aac1",
+	}
```

Comment:
this is already tested in `artifacts_test.go`, I think we can revert to before.

Reply:
I added these tests to make sure that the extraction from checksums.txt is correct,
i.e. in case it'll be e.g. `artifact-name  sha-val` instead of `sha-val artifact-name`,
and tbh I'm not sure it's covered in the other tests.

anyway, added a commit that does just that. I hope that I understood correctly and that it's the correct place to check it.